website/src/app/api/newsletter/subscription/public/route.ts

```
@@ -1,20 +1,44 @@
+type CreateNewsletterSubscriptionRequest = { json(): Promise<CreateNewsletterSubscription> } & Request;
 
-export async function POST(request: CreateNewsletterSubscriptionReqeust) {
+type SendgridClientProps = {
+	SENDGRID_API_KEY: string;
+	SENDGRID_LIST_ID: string;
+	SENDGRID_SUPPRESSION_LIST_ID: number;
+};
+
+const validateSendgridClientProps = (): SendgridClientProps => {
+	const suppressionListId = parseInt(process.env.SENDGRID_SUPPRESSION_LIST_ID!);
+	if (isNaN(suppressionListId)) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)